twins/mongodb/states.go

```diff
@@ -56,7 +56,7 @@ func (sr *stateRepository) Update(ctx context.Context, st twins.State) error {
 func (sr *stateRepository) Count(ctx context.Context, tw twins.Twin) (int64, error) {
 	coll := sr.db.Collection(statesCollection)
 
-	filter := bson.D{{"twinid", tw.ID}}
+	filter := bson.M{"twinid": tw.ID}
```

Comment:
shouldn't `"twinid"` be constant?

Reply:
`twinid` is the string key name of the field in the bson. Therefore, i give it here as is.